.dockerignore

```diff
@@ -12,29 +10,34 @@ report.*.json
 # Rust
 packages/backend/native-utils/target
 
-# Coverage
+# Developer stuff
+.git
+Dockerfile
+docker-compose.yml
 coverage
+chart
+doc
+dev
+npm-debug.log
+*.pem
+run.bat
+api-docs.json
+*.log
+*.code-workspace
+*.md
 
 # config
 /.config
+ormconfig.json
 
 # misskey
 built
 db
 elasticsearch
 redis
-npm-debug.log
-*.pem
-run.bat
-api-docs.json
-*.log
-*.code-workspace
 .DS_Store
 files
-ormconfig.json
-packages/backend/assets/instance.css
 
-# dockerignore custom
-.git
-Dockerfile
-docker-compose.yml
+# I don't see why this should be ignored - m.
```

Comment:
that's basically something for the admins of each instance to write custom css just for their instance, so it shouldn't be version-controlled, etc